BusyList/Parsing/CommandGrammar.cs

```diff
@@ -23,6 +24,15 @@ public static class CommandGrammar
         private static readonly Parser<int> _number =
             Parse.Number.Select(s => int.Parse(s));
 
+        private static readonly Parser<string> _priority =
```

Comment:
While this certainly works, see if you can't use [the Sprache Return command](https://justinpealing.me.uk/post/2020-04-06-sprache5-select-and-return/#return) to clean this up.

Reply:
If you need help, reach out to me on discord and we'll figure it out.